cmake/InstallConfigurations.cmake

```diff
@@ -0,0 +1,40 @@
+if (WIN32 AND SHARED)
+    install(
+            TARGETS raylib
+            ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+            LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+            RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}"
+            PUBLIC_HEADER DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}"
+    )
+else ()
+    install(
+            TARGETS raylib
+            LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+            PUBLIC_HEADER DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}"
+    )
+endif ()
+
+# PKG_CONFIG_LIBS_PRIVATE is used in raylib.pc.in
+if (STATIC)
+    include(LibraryPathToLinkerFlags)
+    library_path_to_linker_flags(__PKG_CONFIG_LIBS_PRIVATE "${LIBS_PRIVATE}")
+    set(PKG_CONFIG_LIBS_PRIVATE ${__PKG_CONFIG_LIBS_PRIVATE} ${GLFW_PKG_LIBS})
+    string(REPLACE ";" " " PKG_CONFIG_LIBS_PRIVATE "${PKG_CONFIG_LIBS_PRIVATE}")
+elseif (SHARED)
+    set(PKG_CONFIG_LIBS_EXTRA "")
+    set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_LIBDIR}")
+    set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+    set(CMAKE_MACOSX_RPATH ON)
```

Comment:
```suggestion
```
According to CMake those values are used to initialize the target properties. Since the target is already created those values do nothing here. (Or they belong above the add_library call. )

Reply:
@Neumann-A Thanks for spotting this. They were after the target initialization in the previous version of the file so I would gadly remove them.